logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/ConversationEventReceiver.kt

```diff
@@ -376,6 +374,35 @@ class ConversationEventReceiverImpl(
         }
     }
 
+    private suspend fun handleDeleteMessage(
+        content: MessageContent.DeleteMessage,
+        message: Message
+    ) {
+        if (isSenderVerified(content.messageId, message.conversationId, message.senderUserId)) {
+            messageRepository.getMessageById(message.conversationId, content.messageId)
+                .onSuccess { messageToRemove ->
+                    (if (messageToRemove.content is MessageContent.Asset)
+                        (messageToRemove.content as MessageContent.Asset).value.remoteData else null)
```

Comment:
same here safe case operator can be used `as?`

Reply:
✅